Package.swift

```diff
@@ -22,6 +22,10 @@ let package = Package(
       name: "LambdaRuntime",
       dependencies: ["AsyncHTTPClient", "NIO", "NIOHTTP1", "NIOFoundationCompat", "Logging", "Base64Kit"]
     ),
-    .testTarget(name: "LambdaRuntimeTests", dependencies: ["LambdaRuntime", "NIOTestUtils", "Logging"])
+    .target(
+      name: "LambdaRuntimeTestUtils",
+      dependencies: ["LambdaRuntime"]
```

Comment:
we need to add `NIOHTTP1` here, since we use `HTTPHeaders`

Reply:
Of course!!